docs/concepts/overview/components.md

```diff
@@ -59,74 +59,69 @@ The following controllers have cloud provider dependencies:
   * Service Controller: For creating, updating and deleting cloud provider load balancers
   * Volume Controller: For creating, attaching, and mounting volumes, and interacting with the cloud provider to orchestrate volumes
 
-### addons
+## Node Components
 
-Addons are pods and services that implement cluster features. The pods may be managed
-by Deployments, ReplicationControllers, and so on. Namespaced addon objects are created in
-the `kube-system` namespace.
+Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
 
-Addon manager creates and maintains addon resources. See [here](http://releases.k8s.io/HEAD/cluster/addons) for more details.
+### kubelet
 
-#### DNS
+{% glossary_definition term_id="kubelet" length="all" %}
 
-While the other addons are not strictly required, all Kubernetes clusters should have [cluster DNS](/docs/concepts/services-networking/dns-pod-service/), as many examples rely on it.
-
-Cluster DNS is a DNS server, in addition to the other DNS server(s) in your environment, which serves DNS records for Kubernetes services.
+### kube-proxy
 
-Containers started by Kubernetes automatically include this DNS server in their DNS searches.
+[kube-proxy](/docs/admin/kube-proxy/) enables the Kubernetes service abstraction by maintaining
+network rules on the host and performing connection forwarding.
 
-#### Web UI (Dashboard)
+### Container Runtime
 
-[Dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard/) is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage and troubleshoot applications running in the cluster, as well as the cluster itself.
+Container Runtime is a software that is responsible for running containers.
```

Comment:
I don't think Container Runtime is a proper name. So this should probably be, "The container runtime is the software that is responsible ...".

Reply:
Done